CLAUDE.md

```
@@ -0,0 +1,2 @@
+Specific rules for Pelela Project:
+- Refer to AGENTS.md for coding standards and persona.
```

markdownlint-cli2 (0.22.1)

[warning] Files should end with a single newline character

(MD047, single-trailing-newline)